Tests/DangerSwiftLintTests/DangerSwiftLintTests.swift

```diff
@@ -10,7 +10,7 @@ class DangerSwiftLintTests: XCTestCase {
     override func setUp() {
         executor = FakeShellExecutor()
         // This is for me, testing. Uncomment if you're running tests locally.
-        FileManager.default.changeCurrentDirectoryPath("/Users/ash/bin/danger-swiftlint")
+        FileManager.default.changeCurrentDirectoryPath("/Users/lukaszmroz/Projects/OtherProjects/Libraries/danger-swiftlint")
```

Comment:
Probably need to revert this guy to the original ?

Reply:
Awww, I was pretty sure I didn't pushed this one, nice catch!